README.md

```diff
@@ -56,6 +56,16 @@ _Context: global_
 Use this command to get a listing of all commands available under `elastic-package` and a brief
 description of what each command does.
 
+### `elastic-package completion`
```

Comment:
Could you please put this command in alphanum. order?

Reply:
I did so, but it got overwritten by running `make update-readme` (which I had to run because the CI was red).

As `completion` is not a command we implemented, I moved it near `help` for this reason.

How may I overcome this issue?

Reply:
Ah, you're right. We generate all these command infos.